.pre-commit-config.yaml

```
@@ -1,7 +1,7 @@
 repos:
 
-  - repo: https://github.com/charliermarsh/ruff-pre-commit
-    rev: "v0.0.264"
+  - repo: https://github.com/astral-sh/ruff-pre-commit
+    rev: "v0.2.1"
```

I think there's no reason why we can't use ruff-pre-commit's latest version:

```suggestion
    rev: "0.12.0"
```